gtsam/hybrid/tests/Switching.h

```diff
@@ -44,33 +47,28 @@ using symbol_shorthand::X;
  * system which depends on a discrete mode at each time step of the chain.
  *
  * @param n The number of chain elements.
- * @param keyFunc The functional to help specify the continuous key.
- * @param dKeyFunc The functional to help specify the discrete key.
+ * @param x The functional to help specify the continuous key.
+ * @param m The functional to help specify the discrete key.
  * @return HybridGaussianFactorGraph::shared_ptr
  */
 inline HybridGaussianFactorGraph::shared_ptr makeSwitchingChain(
-    size_t n, std::function<Key(int)> keyFunc = X,
-    std::function<Key(int)> dKeyFunc = M) {
+    size_t n, std::function<Key(int)> x = X, std::function<Key(int)> m = M) {
```

Comment:
Should we just kill x and m and use X and M directly?

Reply:
Sure, done

Reply:
Turns out this functionality was actually used in testHGFG